prompts/litellm/google-gemini-research.yml

```
@@ -0,0 +1,46 @@
+  Focus areas: {focus_areas}
+  
+  Please structure your response with:
+  
+  ## Overview
+  Brief introduction to the topic
+  
+  ## Key Points
+  - Main findings and facts
+  - Important statistics or data
+  - Current trends
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)